tcs/csp_solver_core.h

```diff
@@ -388,34 +388,29 @@ class C_csp_collector_receiver
 	
 	struct S_csp_cr_solved_params
 	{
-		double m_T_htf_cold_des;		//[K]
-		double m_P_cold_des;			//[kPa]
-		double m_x_cold_des;			//[-]
-        double m_T_htf_hot_des;         //[K]
-		double m_q_dot_rec_des;			//[MW]
-		double m_A_aper_total;			//[m^2] Total solar field aperture area
-        double m_dP_sf;                 //[bar] Total field pressure drop
-
-		S_csp_cr_solved_params()
-		{
-			m_T_htf_cold_des = m_P_cold_des = m_x_cold_des = m_T_htf_cold_des =
-				m_q_dot_rec_des = m_A_aper_total = m_dP_sf = std::numeric_limits<double>::quiet_NaN();
-		}
+		double m_T_htf_cold_des =		//[K]
+		  std::numeric_limits<double>::quiet_NaN();
+		double m_P_cold_des =			//[kPa]
+		  std::numeric_limits<double>::quiet_NaN();
+		double m_x_cold_des =			//[-]
+		  std::numeric_limits<double>::quiet_NaN();
+	  	double m_T_htf_hot_des =         	//[K]
+		  std::numeric_limits<double>::quiet_NaN();
+		double m_q_dot_rec_des =		//[MW]
+		  std::numeric_limits<double>::quiet_NaN();
+		double m_A_aper_total = 		//[m^2] Total solar field aperture area
+		  std::numeric_limits<double>::quiet_NaN();
+	  	double m_dP_sf =                 	//[bar] Total field pressure drop
+		  std::numeric_limits<double>::quiet_NaN();
 	};
 
 	struct S_csp_cr_inputs
 	{	
-		double m_field_control;			//[-] Defocus signal from controller (can PC and TES accept all receiver output?)
-		int m_input_operation_mode;		//[-]
-        double m_adjust;                //[-] Field availability / adjustment factor
-
-		S_csp_cr_inputs()
-		{
-			m_field_control = std::numeric_limits<double>::quiet_NaN();
-			m_adjust = std::numeric_limits<double>::quiet_NaN();
-
-			m_input_operation_mode = -1;
-		}
+		double m_field_control =		//[-] Defocus signal from controller (can PC and TES accept all receiver output?)
+		  std::numeric_limits<double>::quiet_NaN();
+	  	int m_input_operation_mode = 1;		//[-]
```

Comment:
Please change to -1.

Reply:
All of `double` class members previously assigned to quiet_NaN()?

Reply:
Done.

Reply:
No, sorry, I should have been more clear. The doubles were good as quiet_NaN's, you just missed the negative sign with the m_input_operation_mode as it was originally set to -1.

Reply:
Oh, gotcha.

Reply:
Sorry for the mixup. :smile: